content/en/blog/2019/webhook/index.md

```diff
@@ -37,4 +37,4 @@ will not be able to alter the webhook configurations.
 and that the certificate chain used by the webhook server is valid. This reduces the errors
 that can occur before a server is ready or if a server has invalid certificates.
 
-To try this new feature, refer to the [Istio webhook management task](https://archive.istio.io/1.4/docs/tasks/security/webhook).
\ No newline at end of file
+To try this new feature, refer to the [Istio webhook management task](https://archive.istio.io/v1.4/docs/tasks/security/webhook).
```

Comment:
It's good that it caught broken links like this, but I'm wondering why it did. I think that the reason that htmlproofer missed it was because we have explicitly disabled checking of links to various sites, and most recently to external sites in general, because we have too many broken builds caused by temporary outages or other changes in sites we have no control over. I'm wondering if linkinator has an equivalent feature (ignore links to certain sites, or external sites in general). If not, we may end up being frustrated again by regular build failures.

Reply:
yes, if you look at lint_site.sh:183, we are ignoring the regex "^((?!localhost).)*$". This essentially ignores any site that isn't local. The new code is still respecting the CHECK_EXTERNAL_LINKS variable